Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.h

```diff
@@ -16,6 +16,13 @@ namespace ROS2
     //! A structure capturing configuration of a single camera sensor with up to two image sources (color and depth).
     struct CameraSensorConfiguration
     {
+        enum class CameraInfoPublisherType
+        {
+            None,
+            Single,
+            Separate
```

Comment:
`None` option does not make much sense.
I am not sure if there is any benefit in not publishing as "Separate" always.